app/components/concept-page/concept-completed-modal.ts

```
@@ -0,0 +1,44 @@
+    }
+
+    return '/catalog';
+  }
+
+  @action
+  handleSignInButtonClick() {
+    // We redirect to `<backend_url>/concepts/:id/mark_as_complete` to ensure the user's progress is saved.
+    const markAsCompleteUrl = new URL(`${config.x.backendUrl}/concepts/${this.args.concept.id}/mark_as_complete`);
+
+    markAsCompleteUrl.searchParams.set('redirect_url', `${window.origin}${this.redirectPathAfterLogin}`);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)